qiita_ware/ebi.py

```diff
@@ -981,3 +950,52 @@ def send_xml(self):
             print "FAILED"
 
         return (study_accession, submission_accession)
+
+    def generate_demultiplexed_fastq(self):
+        """Generates demultiplexed fastq
+
+        Returns
+        -------
+        demux_samples
+            List of successful demultiplexed samples
+
+        Notes
+        -----
+        - As a performace feature, this method will check if self.ebi_dir
```

Comment:
This seems like a fine performance feature, but what would be a way to bypass this? i.e. a case where you know that there's something wrong with the generate per-sample FASTQ files and (luckily) your submission failed before? Maybe I am missing something but it seems like the only solution would be to delete the files manually?